java-frontend/src/main/java/org/sonar/java/se/constraint/Constraint.java

```diff
@@ -19,6 +19,8 @@
  */
 package org.sonar.java.se.constraint;
 
+import org.sonar.java.se.symbolicvalues.RelationalSymbolicValue;
```

Comment:
In terms of design this is bothering me... I don't think constraints should know about symbolic values. Probably not be modified right now.